nntrainer/compiler/tflite_opnode.cpp

```diff
@@ -42,11 +42,23 @@ void TfOpNode::setLayerNode(const LayerNode &layer) {
     }
     return v;
   };
+  auto create_variables_with_NCHW_to_NHWC = [](auto tensor_getter,
+                                               unsigned size) {
+    Variables v;
+    v.reserve(size);
+    for (unsigned i = 0; i < size; ++i) {
+      Tensor *tensor = const_cast<Tensor *>(tensor_getter(i));
+      tensor->reshape(TensorDim{tensor->batch(), tensor->height(),
```

Comment:
@lhs8928 These tensors are not constant. Reshaping seems enough. Please let me know if I missed something.

Reply:
This needs to be reshape. There is no need to transpose the whole tensor for input and output. Because it will be over written. But we need the transpose for the weight tensor.